src/components/Input/example.js

```diff
@@ -1,34 +1,48 @@
 import React from 'react';
+import styled from 'styled-components';
 
 import { Example, Info } from '../../utils/example';
 
 import Input from '.';
 
 const onChange = () => null;
 
+const StyledExample = styled(Example)`
+  width: 250px;
+`;
+
 export default function InputExample() {
   return (
     <div>
-      <Example>
+      <StyledExample>
         <Info>Plain inputs</Info>
         <Input
           label="Username"
           placeholder="your name here"
           onChange={onChange}
         />
         <Input label="Email" value="ron@hogwarts.edu" onChange={onChange} />
-      </Example>
-      <Example>
+      </StyledExample>
+      <StyledExample>
         <Info>With validation</Info>
         <Input
           label="Email"
-          value="invalid-email"
+          value="invalid-email-that is really long"
```

Comment:
Would be nice to actually make this one editable so that we can test the icon padding interactively (also, it's less confusing when it's editable).